src/com/owncloud/android/files/services/FileDownloader.java

```diff
@@ -626,4 +682,32 @@ private void sendBroadcastNewDownload(DownloadFileOperation download, String lin
         sendStickyBroadcast(added);
     }
 
+    /**
+     * Remove downloads of an account
+     * @param account
+     */
+    private void cancelDownloadsForAccount(Account account){
```

Comment:
OK, thinking twice: there's too much processing here. It's much easier implementing a new method ```IndexedForest.remove(Account)``` and call it from here. Let's deal with it offine.